OracleHTTPServer/dockerfiles/12.2.1.4.0/container-scripts/provisionOHS.sh

```diff
@@ -96,23 +100,58 @@ if [ -z "$NM_PASSWORD" ]; then
    exit
 fi
     
-wlst.sh -skipWLSModuleScanning -loadProperties $PROPERTIES_FILE /u01/oracle/container-scripts/create-sa-ohs-domain.py
+wlst.sh -skipWLSModuleScanning -loadProperties $PROPERTIES_FILE /u01/oracle/create-sa-ohs-domain.py
 # Set the NM username and password in the properties file
-echo "username=$NM_USER" >> /u01/oracle/ohssa/user_projects/domains/ohsDomain/config/nodemanager/nm_password.properties
-echo "password=$NM_PASSWORD" >> /u01/oracle/ohssa/user_projects/domains/ohsDomain/config/nodemanager/nm_password.properties
-mv /u01/oracle/container-scripts/helloWorld.html ${ORACLE_HOME}/user_projects/domains/ohsDomain/config/fmwconfig/components/OHS/ohs1/htdocs/helloWorld.html
+echo "username=$NM_USER" >> ${DOMAIN_HOME}/config/nodemanager/nm_password.properties
+echo "password=$NM_PASSWORD" >> ${DOMAIN_HOME}/config/nodemanager/nm_password.properties
+mv /u01/oracle/helloWorld.html ${DOMAIN_HOME}/config/fmwconfig/components/OHS/$OHS_COMPONENT_NAME/htdocs/helloWorld.html
+
+echo "Copying Configuration to OHS Instance"
+cp  -L /u01/oracle/config/moduleconf/*.conf ${DOMAIN_HOME}/config/fmwconfig/components/OHS/$OHS_COMPONENT_NAME/moduleconf && find ${DOMAIN_HOME}/config/fmwconfig/components/OHS/$OHS_COMPONENT_NAME/moduleconf -print0 -name '.*' | xargs rm -rf
+
+conf=$(ls -l /u01/oracle/config/httpd/*.conf 2>/dev/null | wc -l)
+if [ $conf -gt 0 ]
```

Comment:
Even if there are no files there would be 1 line; so I think it should be compared with 1.

Reply:
greater than 0 is 1 so I prefer not to change it